gaiku-common/src/texture.rs

```diff
@@ -31,6 +31,9 @@ pub trait Texturify2d: Clone + std::fmt::Debug {
   fn get_pixel(&self, x: u32, y: u32) -> Option<[u8; 4]>;
   fn height(&self) -> u32;
   fn len(&self) -> usize;
+  fn is_empty(&self) -> bool {
+    self.len() == 0
+  }
```

Comment:
Not sure if is needed, we need to work out in not allowing the initialization of an empty texture, maybe we should review the texture implementation and address that, but not in this PR.

Reply:
This was required by clippy. Wherever there is a `len` method it makes this suggestion. If we don't want it I will have to add a clippy allow directive instead.

Reply:
I have reverted and added an `#[allow(clippy::len_without_is_empty)]` instead